ingest-app/src/cmr/ingest/validation/business_rule_validation.clj

```diff
@@ -89,11 +82,47 @@
         (when (seq search-errors)
           search-errors)))))
 
+(defn standard-product-is-nasa-provider-validation
+  "Validate if the standard product is true then the provider must be a NASA provider - meaning 
+  that the provider's consortium values must include EOSDIS. If validation fails an error is thrown."
+  [context concept _]
+  (let [provider-id (:provider-id concept)
+        collection (:umm-concept concept)
+        standard-product (:StandardProduct collection)
+        consortiums-str (some #(when (= provider-id (:provider-id %)) (:consortiums %))
+                              (mdb/get-providers context))
+        consortiums (when consortiums-str
+                      (remove empty? (string/split (string/upper-case consortiums-str) #" ")))]
+    (when (and (= true standard-product)
+               (not (some #(= "EOSDIS" %) consortiums)))
+      [(format (str "Standard product validation failed: "
+                    "Standard Product designation is only allowed for NASA data products. This collection is "
+                    "being ingested using a non-NASA provider which means the record is not a NASA record. "
+                    "Please remove the StandardProduct element from the record. "
+                    "The consortium designations include the following [%s] and EOSDIS is not one of them.")
```

Comment:
Would suggest wording slightly differently if this is something the user can address by including the EOSDIS consortium. Such as "Standard Product designation is only allowed for NASA data products. To be designated as a NASA record and Standard Product this collection must include EOSDIS in its current list of consortiums: %s"

If the user can't fix this because its done at the provider level we can probably cut some of the message.

Reply:
The curator generally does not know about consortiums.  Also consortiums are part of the provider and not included in the metadata. Adding/defining consortiums is something that a CMR operator must do. I was just trying to add that into the message so that a CMR operator would know what to do.

Reply:
Ok, I got rid of the last sentence.